test/patch-2.js

```diff
@@ -109,10 +133,7 @@ describe('PATCH', function () {
         .end(function (err, res, body) {
           assert.equal(
             read('sampleContainer/prefixSparql.ttl'),
-            '@prefix schema: <http://schema.org/>.\n' +
-            '@prefix profile: <http://ogp.me/ns/profile#>.\n' +
-            '\n' +
-            '   <#> profile:first_name "Timothy"; a schema:Person .\n')
+            '@prefix : </prefixSparql.ttl#>.\n@prefix schema: <http://schema.org/>.\n@prefix pro: <http://ogp.me/ns/profile#>.\n\n: a schema:Person; pro:first_name "Timothy".\n\n')
```

Comment:
A bit confused about this change given the one above. Let's be consistent with formatting of data. I think the previous change is much more readable.